pandas/core/groupby/groupby.py

```diff
@@ -207,7 +207,7 @@ class providing the base-class of operations.
 See Also
 --------
 Series.pipe : Apply a function with arguments to a series.
-DataFrame.pipe: Apply a function with arguments to a dataframe.
+DataFrame.pipe : Apply a function with arguments to a dataframe.
 apply : Apply function to each group instead of to the
```

Comment:
I don't think we leave a space before the colon in the See Also section (if we do, the rest of changes are wrong)